### PR TITLE
fix(e2e): Let the subscription and related resource reconcile

### PR DESCRIPTION
**Pull request by navidshaikh**
sleep for 5 seconds after subscription create and update in e2e

### CODE REVIEW

File: lib/test/subscription.go [hunk 1]
```diff
@@ -18,6 +18,7 @@ package test
 
 import (
```
Comment (mattmoor) [inline lib/test/subscription.go:19]: Format Go code:
```suggestion
import (
	"time"

```

File: lib/test/subscription.go [hunk 2]
```diff
@@ -18,6 +18,7 @@ package test
...
 	"gotest.tools/assert"
+	"time"
```
Comment (mattmoor) [inline lib/test/subscription.go:21]: Format Go code:
```suggestion

```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `gofmt -s -w $(find -path './vendor' -prune -o -path './third_party' -prune -o -name '*.pb.go' -prune -o -type f -name '*.go' -print)`
  `goimports -w $(find -name '*.go' | grep -v vendor | grep -v third_party | grep -v .pb.go | grep -v wire_gen.go)`

**Review by rhuss — APPROVED** [review]
/lgtm

thanks !

**Pull request merged**